frontend/src/components/Loading.tsx

```diff
@@ -1,12 +1,9 @@
-interface Props {
-  size?: number;
-}
+interface Props {}
 
 export default function Loading(props: Props) {
-  const SIZE = props.size ? props.size : 4;
   return (
-    <span
-      className={`w-${SIZE} h-${SIZE} border-b-2 border-white-900 rounded-full animate-spin mr-5`}
-    ></span>
+    <div
+      className={`w-4 h-4 border-b-2 border-white-900 rounded-full animate-spin mr-5`}
```

Comment:
Since you don't use the dynamic data just use qoutes
```suggestion
      className='w-4 h-4 border-b-2 border-white-900 rounded-full animate-spin mr-5'
```